docs/composer/api-modification.md

```diff
@@ -2,7 +2,7 @@
 
 If you want to modify automatically generated API, you can use composer custom `onRoute` hook.
 
-#### `addComposerOnRouteHook(openApiPath, methods, handler)`
+### addComposerOnRouteHook(openApiPath, methods, handler)
```

Comment:
I actually prefer if functions have the backticks

Reply:
Okay, but can I ask why?

Between this is done.